paddle/fluid/operators/listen_and_serv_op.cc

```diff
@@ -129,6 +129,8 @@ class ListenAndServOp : public framework::OperatorBase {
       }
       if (exit_flag) {
         rpc_service_->ShutDown();
+        rpc_service_->SetCond(1);
```

Comment:
Why need this condition?

Reply:
If not, send op wouldn't exit, it will hang on `Get` operation.